src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallManager.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Deployment.DotNet.Releases;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.NET.Sdk.WorkloadManifestReader;
+using Product = Microsoft.DotNet.Cli.Utils.Product;
+
+namespace Microsoft.DotNet.Workloads.Workload.Install
+{
+    internal class WorkloadInstallManager
```

Comment:
https://blog.codinghorror.com/i-shall-call-it-somethingmanager/

we should give it a better name and put it in spec. We have workload install, workload install manager(aka the caller of "workload install"), it is a reflection of we don't have a clear cut on the abstraction.

Reply:
Could this logic just be moved into the `WorkloadInstallCommand` class?  Then we wouldn't need a separate "manager".